inc/task.class.php

```diff
@@ -538,7 +533,8 @@ protected function buildMqttMessage($policiesToApply) {
          );
          if ($policyMessage === false) {
             // There is an error while applying the policy, continue with next one for minimal impact
-            // TODO: log something
+            Toolbox::logInFile('plugin_flyvemdm_inventory',
+               "Policy value '" . $policyToApply['value'] . "' not applied\n");
```

Comment:
ok to implement logging, but maybe should we add more details : which policy ? which fleet ?

We don't have the fleet object here. Maybe introducing an exception would be better. (and add try / catch block at the level where the  fleet instance is available